generated/nifake/nifake.proto

```diff
@@ -1,6 +1,6 @@
 
 //---------------------------------------------------------------------
-// This file is generated from NI-FAKE API metadata version 23.0.0
+// This file is generated from NI-FAKE API metadata version 23.0.0d96
```

Comment:
Note: NI-FAKE hapigen metadata has changed some since we last updated here, but that shouldn't really matter. It still builds and the Unit/Integration tests still pass which is all we care about.